andela/ah-fulldeck-frontend
# #162948947 Setup React Redux

**What does this PR do?**

* Setup React/Redux

**Description of the tasks to be completed?**

* Create a file structure for Redux workflow.
* Add packages `redux, react-redux, redux-thunk`
* Add dev dependencies `redux-immutable-state-invariant, redux-logger, redux-devtools-extension`
* Create the Redux store

#### How should this be manually tested?
* Git clone this repo: `git clone https://github.com/andela/ah-fulldeck-frontend.git`
* Git checkout to ch-setup-redux-162948947
* Make sure have the redux dev tool extension
* npm start :-This should open up a browser tab
* Click on the redux dev tools icon on the browser
* This should open up the empty redux store - make sure it looks like the screenshot below.

#### What are the relevant pivotal tracker stories?
[162948947](https://www.pivotaltracker.com/story/show/162948947)

#### Screenshots
<img width="1280" alt="screenshot 2019-02-07 at 09 20 36" src="https://user-images.githubusercontent.com/22760022/52394773-9dbae400-2abc-11e9-9237-241e4a0414f1.png">
